package.json

```diff
@@ -18,17 +18,53 @@
   "author": "CAR FOR YOU",
   "license": "MIT",
   "homepage": "https://github.com/carforyou/carforyou-eslint-rules-pkg#readme",
+  "peerDependencies": {
```

Comment:
What was your basis to elaborate on what plugins we wanna use?

Reply:
I picked the plugins based on the tooling we use. Most of the extra plugins do not impact our current code by a lot but help us with some minor problems. E.g. I discovered a bunch of tests that were using RTL helpers that were asynchronous in a synchronous way and vice versa.